test/test_custom_ops.py

```diff
@@ -211,6 +211,15 @@ def _(x: torch.Tensor) -> torch.Tensor:
 
         example = torch.zeros([10, 20], device=device)
         torch.library.opcheck(f, args=[example])
+    
+    def test_single_element_tuple_output(self, device):
+        @torch.library.custom_op("test::add", mutates_args=[])
+        def add(x: torch.Tensor, y: torch.Tensor) -> Tuple[torch.Tensor]:
+            return (x.clone(),)
+        
+        x = torch.randn(3, device=device)
+        ret = add(x, x)
+        self.assertTrue(torch.equal(x, ret))
```

Comment:
nit: use self.assertEquals(x, ret).

Also, torch.equal is actually "bitwise equality" -- for floating point numbers we usually check equality via checking if they're "close enough". That's what torch.allclose does and what self.assertEquals does for floating point values

Reply:
feedback addressed.